apps/io-wallet-user-func/src/infra/http/handlers/__test__/create-wallet-instance.spec.ts

```diff
@@ -72,6 +82,15 @@ describe("CreateWalletInstanceHandler", () => {
       }),
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const mockEmailNotificationService: any = {
```

Comment:
you should never use `any` like this (even in tests) and lint rules should only be disabled on few occasions

Reply:
Aaargh my bad habit!